x-pack/auditbeat/module/system/packages/_meta/fields.yml.disabled

```diff
@@ -0,0 +1,50 @@
+- name: packages
+  type: group
+  description: >
+    `packages` contains information about installed packages.
+  release: experimental
+  fields:
+  - name: package
+    type: array
```

Comment:
I'm not sure if that has the affect you are hoping for. Do you need this data indexed?

Reply:
The `packages` metricset is disabled - note this file's name is `fields.yml.disabled`.

Reply:
if it's disable, is it worth backporting?

Reply:
It's more effort to not backport it :) but yes, I can remove it